backend/functions/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "outDir": "lib",
     "sourceMap": true,
     "strict": true,
-    "target": "es2019",
+    "target": "es2020",
```

Comment:
?

Reply:
> TypeScript target changed to es2020 so I can use [Promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)

from PR description, does that make sense